### PR TITLE
Issue 2 view an entry

### PR DESCRIPTION
**Pull request by restropiajr**
Recording 2023-05-30 at 17 17 37

### CODE REVIEW

**Comment by restropiajr** [comment]
Recording 2023-05-30 at 18 28 30

File: index.html [hunk 1]
```diff
@@ -24,13 +24,15 @@
       <div class="row">
         <div class="column-full">
           <h1>Code Journal</h1>
+          <a id="entries-button">Entries</a>
```
Comment (shawnkost) [inline index.html]: Needs an `href`. You can just set it to a `#`

File: index.html [hunk 2]
```diff
@@ -58,7 +60,19 @@ <h2>New Entry</h2>
       </div>
     </div>
 
-    <div data-view="entries" class="hidden"></div>
+    <div class="entries" data-view="entries">
```
Comment (shawnkost) [inline index.html]: This should have a class of `"hidden"` so that it's hidden by default

File: js/main.js [hunk 3]
```diff
@@ -25,6 +23,90 @@ $form.addEventListener('submit', event => {
...
+
+  const columnHalfDiv = document.createElement('div');
+  columnHalfDiv.className = 'column-half';
+  rowDiv.appendChild(columnHalfDiv);
+
+  const h3 = document.createElement('h3');
+  h3.textContent = entry.title;
+  columnHalfDiv.appendChild(h3);
+
+  const p = document.createElement('p');
```
Comment (shawnkost) [inline js/main.js]: Don't forget to include the `$` before the variable names. We are creating DOM elements, so we should make sure to include that `$`!

File: js/main.js [hunk 4]
```diff
@@ -25,6 +23,90 @@ $form.addEventListener('submit', event => {
...
+  }
+}
+
+// View Swap Function
+const entryFormview = document.querySelector('[data-view="entry-form"]');
+const entriesView = document.querySelector('[data-view="entries"]');
+function viewSwap(view) {
+
+  entryFormview.classList.add('hidden');
+  entriesView.classList.add('hidden');
```
Comment (shawnkost) [inline js/main.js]: You can do this, but why hide both of them by default? Why not just hide them each individually in your conditional? If the view were to stay the same, we don't want to hide the page, we'd just want to keep it visible

**Review by shawnkost — CHANGES_REQUESTED** [review]
Nicely done Roland! A few changes requested here

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**